packages/contracts/test/voting/token-voting.ts

```diff
@@ -375,47 +405,55 @@ describe('TokenVoting', function () {
     });
 
     it('should not double-count votes by the same address', async () => {
-      await advanceTimeTo(startDate);
+      await advanceIntoVoteTime(startDate, endDate);
 
       await governanceErc20Mock.mock.getPastVotes.returns(1);
 
-      // yes still ends up to be 1 here even after voting
-      // 2 times from the same wallet.
       await voting.vote(id, VoteOption.Yes, false);
       await voting.vote(id, VoteOption.Yes, false);
       expect((await voting.getProposal(id)).yes).to.equal(1);
+      expect((await voting.getProposal(id)).no).to.equal(0);
+      expect((await voting.getProposal(id)).abstain).to.equal(0);
 
-      // yes gets removed, no ends up as 1.
       await voting.vote(id, VoteOption.No, false);
       await voting.vote(id, VoteOption.No, false);
+      expect((await voting.getProposal(id)).yes).to.equal(0);
       expect((await voting.getProposal(id)).no).to.equal(1);
+      expect((await voting.getProposal(id)).abstain).to.equal(0);
 
-      // no gets removed, abstain ends up as 1.
```

Comment:
These comments are helpful IMO

Reply:
Not a big deal, really... just a general comment that reading one sentence may save us 2 minutes of reading code in the future. But anyway, testing code is not the target, but rather the helper